indico_hub/server.py

```diff
@@ -88,16 +88,16 @@ def update_instance(uuid, **kwargs):
     inst = Instance.query.filter_by(uuid=uuid).first()
     if inst is None:
         abort(404, description='BAD REQUEST: user already exist')
-    if(kwargs['enabled'] == False):
-        #unregister instance
+    if kwargs['enabled'] is False:
+        # unregister instance
         db.session.delete(inst)
```

Comment:
better to just mark the instance as inactive (new bool column) instead of deleting it from the db

Reply:
hmmm should I just save 'enabled' as false?
I think this will do it